client/styles/components/component_kit/buttons.scss

```diff
@@ -45,6 +45,75 @@
     }
 }
 
+.gradient-button-label {
+    padding-top: 6px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.GradientButton {
+    min-width: 70px;
+    height: 40px;
+    padding: 1px;
+    font-weight: 600;
+    margin: 0 7px;
+    color: #4a4a4a;
+    font-weight: 500;
+    border: 1px double transparent;
+    --angle: 45deg;
+    border-radius: 10px;
```

Comment:
We should inherit or use the same old border radius from the others currently implemented.